sharder/sharded-bot.js

```diff
@@ -100,6 +103,22 @@ const manager = new ShardingManager(startup, {
   token,
 });
 
-manager.on('shardCreate', (shard) => console.log(`Shard ${shard.id} launched`));
+manager.on('shardCreate', (shard) => {
+  shard.on('reconnecting', () => {
+    console.log(`Shard [${shard.id}] is reconnecting`);
+  });
+  shard.on('spawn', () => {
+    console.log(`Shard [${shard.id}] spawned`);
+  });
+  shard.on('ready', () => {
+    console.log(`Shard [${shard.id}] is ready`);
+  });
+  shard.on('death', () => {
+    console.log(`Shard [${shard.id}] died`);
+  });
+  shard.on('error', (err) => {
+    console.error(`Error in Shard [${shard.id}]: `, err);
+  });
+});
 
-manager.spawn(totalShards, 5500, timeout);
+manager.spawn(totalShards, 15500, timeout).catch((e) => console.log(e));
```

Comment:
These arguments are not being passed correctly, as per [`ShardingManager#spawn`](https://discord.js.org/#/docs/discord.js/13.10.2/class/ShardingManager?scrollTo=spawn)

Additionally, the catch can simply be `.catch(console.error)` (or you can just remove it completely)